Integrations/integration-McAfee-ePO.yml

```diff
@@ -169,22 +177,87 @@ script:
                 }
             }
         }
+        return {
+            Type: entryTypes.note,
+            Contents: readyRes,
+            ContentsFormat: formats.json,
+            HumanReadable: md
+        };
+    }
 
-        return {Type: entryTypes.note, Contents: readyRes, ContentsFormat: formats.json, HumanReadable: md};
-    };
+    function epoGetLatestDatRaw() {
+        var url = "http://update.nai.com/products/commonupdater/gdeltaavv.ini"
+        //Another option is to use avvdat.ini - provided here in case any additional data from that file is needed in the future
+        //var url = "http://update.nai.com/Products/CommonUpdater/avvdat.ini"
+        result = doRawHttp("GET",url);
+        var res = '';
+        var sections = result.Body.split('\r\n\r\n');
+        var lines = sections[0].split('\r\n');
+        for (var i=0;i<lines.length;i++) {
+            if (lines[i].indexOf("CurrentVersion")===0){
+                return lines[i].split('=')[1];
+            }
+        }
+        throw 'Cannot get McAfee latest DAT file version';
+    }
+
+    function epoGetLatestDat() {
+        var datVersion = epoGetLatestDatRaw();
+        var md='McAfee Latest DAT file version is: **' + datVersion + '**\n';
+        ec = {};
+        ec.mcafee = {"latestDAT": datVersion};
+        return {
+            Type: entryTypes.note,
+            Contents: datVersion,
+            ContentsFormat: formats.json,
+            HumanReadable: md,
+            EntryContext: {
+                    "mcafee(val.latestDAT === obj.latestDAT)": ec.mcafee, //backward compatability
+                    "McAfee.ePO.latestDAT": datVersion
+            }
+        };
+    }
+
+    function epoGetCurrentDat() {
+        var pArgs = {"searchText":"VSCANDAT1000"};
+        var result = callEpo('repository.findPackages', pArgs);
+        if (result && result.length>0) {
+            result = result[0];
+            var epoDAT = result.productDetectionProductVersion.split('.')[0];
+            ec = {};
+            ec.mcafee = {"epoDAT": epoDAT};
+            var md = "McAfee ePO Current DAT file version in repository is: **" + epoDAT + '**\n';
+            return {
+                Type: entryTypes.note,
+                Contents: result,
+                ContentsFormat: formats.json,
+                HumanReadable: md,
+                EntryContext: {
+                    "mcafee(val.epoDAT==obj.epoDAT)": ec.mcafee, //backward compatability
+                    "McAfee.ePO.epoDAT": epoDAT
```

Comment:
should there be `McAfee.ePO.epoDAT`= >`McAfee.ePO(val.epoDAT == obj.epoDAT)`?

Reply:
indeed, fixed it.